src/sirhelpers.c

```diff
@@ -196,11 +196,11 @@ bool _sir_validtextcolor(sir_colormode mode, sir_textcolor color) {
             valid = SIRTC_DEFAULT == color || ((color & 0xff000000) == 0);
             break;
         }
-        case SIRCM_INVALID:
```

Comment:
This technically should be removed, since you can't set the color mode to this value, and the default is set on startup, but then we'll get linter warnings about not handling that case.